demo/site/src/layout/header/MobileMenu.tsx

```diff
@@ -0,0 +1,287 @@
+                                                                ))}
+                                                            </PageLayoutContent>
+                                                        </PageLayout>
+                                                    </SubLevelNavigation>
+                                                </FocusLock>
+                                            )}
+                                        </li>
+                                    );
+                                })}
+                            </TopLevelNavigation>
+                        </FocusLock>
+                    </PageLayoutContent>
+                </PageLayout>
+            </MenuContainer>
+        </Root>
+    );
+};
+
+const Root = styled.div`
+    ${({ theme }) => theme.breakpoints.md.mediaQuery} {
+        display: none;
+    }
+`;
+
+const PageLayoutContent = styled.div`
+    grid-column: 2 / -2;
+`;
+
+const MenuButton = styled.button`
+    appearance: none;
+    border: none;
+    background-color: transparent;
+    color: inherit;
+    padding: 0;
+    width: 24px;
+    height: 24px;
+`;
+
+const MenuContainer = styled.div<{ $isMenuOpen: boolean }>`
+    display: block;
+    position: fixed;
+    top: var(--header-height);
+    left: 0;
+    height: 0;
+    width: 100vw;
+    z-index: 40;
+    background-color: ${({ theme }) => theme.palette.gray["200"]};
+    overflow: auto;
+    visibility: hidden;
+    transition:
+        height 0.15s ease-out,
+        visibility 0s linear 0.15s;
+
+    ${({ $isMenuOpen }) =>
+        $isMenuOpen &&
+        css`
+            visibility: visible;
+            height: calc(var(--full-viewport-height) - var(--header-height));
+            transition: height 0.25s ease-in;
+        `}
+`;
+
+const TopLevelNavigation = styled.ol`
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+`;
+
+const SubLevelNavigation = styled.ol<{ $isExpanded: boolean }>`
+    list-style-type: none;
+    position: fixed;
+    top: var(--header-height);
+    left: 0;
+    height: calc(var(--full-viewport-height) - var(--header-height));
+    width: 100vw;
+    background-color: ${({ theme }) => theme.palette.gray["200"]};
+    padding: 0;
+    overflow: auto;
+    visibility: hidden;
+    transform: translateX(100%);
+    transition:
+        transform 0.2s ease-out,
+        visibility 0s linear 0.2s;
+
+    ${({ $isExpanded }) =>
+        $isExpanded &&
+        css`
+            visibility: visible;
+            transform: translateX(0);
+            transition: transform 0.2s ease-in;
+        `}
+`;
+
+const Link = styled(PageLink)`
+    width: 100%;
+    text-decoration: none;
+    display: inline-block;
+    padding: ${({ theme }) => theme.spacing.s500} 0;
+    font-family: ${({ theme }) => theme.fontFamily};
+    color: ${({ theme }) => theme.palette.text.primary};
+
+    ${({ theme }) => theme.breakpoints.sm.mediaQuery} {
```

Comment:
Why don't we set the hover style below sm breakpoint? Is there problem setting it? I did not saw that anywhere before 🤔

Reply:
ahh found it, there was an error on scrollable mobile menus. when the element is touched on scrolling the hover effect triggers 🕵️